src/overlays/actors/ovl_En_Anubice/z_en_anubice.h

```diff
@@ -9,34 +9,54 @@ struct EnAnubice;
 
 typedef void (*EnAnubiceActionFunc)(struct EnAnubice*, struct GlobalContext*);
 
+typedef enum {
+    /*  0 */ ANUBICE_LIMB_NONE,
+    /*  1 */ ANUBICE_LIMB_ROOT,
+    /*  2 */ ANUBICE_LIMB_BODY_ROOT,
+    /*  3 */ ANUBICE_LIMB_CHEST,
+    /*  4 */ ANUBICE_LIMB_ABDOMEN_ROOT,
+    /*  5 */ ANUBICE_LIMB_LOWER_ABDOMEN,
+    /*  6 */ ANUBICE_LIMB_UPPER_ABDOMEN,
+    /*  7 */ ANUBICE_LIMB_TAIL_ROOT,
+    /*  8 */ ANUBICE_LIMB_TAIL_BASE,
+    /*  9 */ ANUBICE_LIMB_TAIL_TIP,
+    /* 10 */ ANUBICE_LIMB_JEWEL_ROOT,
+    /* 11 */ ANUBICE_LIMB_JEWEL,
+    /* 12 */ ANUBICE_LIMB_HEAD_ROOT,
+    /* 13 */ ANUBICE_LIMB_HEAD,
+    /* 14 */ ANUBICE_LIMB_JAW_ROOT,
+    /* 15 */ ANUBICE_LIMB_JAW,
+    /* 16 */ ANUBICE_LIMB_MAX
+} AnubiceLimbs;
+
 typedef struct EnAnubice {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ SkelAnime skelAnime;
-    /* 0x0190 */ Vec3s jointTable[16];
-    /* 0x01F0 */ Vec3s morphTable[16];
+    /* 0x0190 */ Vec3s jointTable[ANUBICE_LIMB_MAX];
+    /* 0x01F0 */ Vec3s morphTable[ANUBICE_LIMB_MAX];
     /* 0x0250 */ EnAnubiceActionFunc actionFunc;
     /* 0x0254 */ s16 timeAlive;
-    /* 0x0256 */ s16 unk_256;
-    /* 0x0258 */ s16 unk_258;
+    /* 0x0256 */ s16 isNearWall; // Technically any bg poly, but in practice usually a wall
```

Comment:
```suggestion
    /* 0x0256 */ s16 isNearWall; // Technically any bg poly, but in practice usually a wall
```
I would avoid using "bg" where we can, and use "collision" instead

Reply:
until when and if we agree to use "collision" to only mean scene/dyna stuff, its prob for the best to be bg so people dont confuse it as including a collider

Reply:
well "collision poly" is non ambiguous imo, that's even the name of a struct involved in collision headers.

Also in some contexts "bg poly" only refers to "static"/"scene" collision iirc, excluding dynapoly, which was more my concern here

but yeah sure this can be cleaned up eventually

Reply:
yeah thats fair. i agree going with collision long term will be for the best, but plenty of people still refer to colliders as being under the "collision" umbrella. at least colloquially. and the file that handles it is called collision_check

we for sure need to clean this up in general at a later time